feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -383,7 +383,6 @@ func configureATE(t *testing.T, top gosnappi.Config, ate *ondatra.ATEDevice, ate
 
 	dev := top.Devices().Add().SetName(Name + ".Dev")
 	eth := dev.Ethernets().Add().SetName(Name + ".Eth")
-	eth.Connection().SetChoice("port_name")
```

Comment:
just curious on this one why is this removed?

Reply:
Basically "eth.SetPortName" is deprecated and it was conflicting with eth.Connection.SetChoice....
Now we have to correct way.